### DISCUSSION TITLE
Why linux and windows generate different open xml?

### DISCUSSION THREAD

**Discussion by rstm-sf** [OfficeDev/Open-XML-SDK#1244] [Q&A]
Left Ubuntu -- Right Windows
![image](https://user-images.githubusercontent.com/2669927/204193895-43994e6b-1426-4b1d-a219-09d22dc37bc3.png)

**Comment by twsouthwick** [comment]
can you provide the code you use to generate? something we can reproduce it

**Reply by rstm-sf** [reply]
I'm about to do a golden test on the [OfficeIMO](https://github.com/EvotecIT/OfficeIMO) repository and getting a no equal https://dev.azure.com/evotecpl/OfficeIMO/_build/results?buildId=2514&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=6278c56e-b9dd-5471-9c3c-f3062039459c&l=46

https://github.com/EvotecIT/OfficeIMO/blob/f87a8eb13547ea337d41bb5b1c7a3c27b5c3fe4f/OfficeIMO.VerifyTests/Word/AdvancedDocumentTests.cs#L21

I wanted to highlight a minimal example, a little later, sorry. 

I don't see any os specific code

<details>
  <summary>Screen search</summary>

![image](https://user-images.githubusercontent.com/2669927/204641629-7f56d4db-cccc-46cc-8593-cf16e7a848bd.png)

</details>

**Comment by twsouthwick** [comment]
There's no os specific code I know of in the SDK. Let us know if you get a repro; if it is in the SDK, feel free to open an issue for it.

**Reply by rstm-sf** [reply]
Ok. I tried highlighting the code but couldn't reproduce https://github.com/rstm-sf/OpenXmlSdkHyperlink

(Really a problem in CI?..)
